test/controllers/categories_controller_test.rb

```diff
@@ -10,35 +10,81 @@
     
     it "gives back a 404 if there are no works available" do
```

Comment:
I think you meant to say "it returns with a success if there are no works available"? Otherwise, the "must_respond_with :success" contradicts this statement.